kitsune/products/utils.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import Q
+
+from kitsune.products.models import Banner
+
+
+def cta_banner(products=[], topics=[], page_type="", request=None):
```

Comment:
I would make `request` the first arg. This is not a view but it helps with consistency